src/libstore/build/worker.cc

```diff
@@ -283,9 +283,14 @@ void Worker::run(const Goals & _topGoals)
                             "or enable remote builds."
                             "\nhttps://nixos.org/manual/nix/stable/advanced-topics/distributed-builds.html");
                 else
-                   throw Error("unable to start any build; remote machines may not have "
-                            "all required system features."
-                            "\nhttps://nixos.org/manual/nix/stable/advanced-topics/distributed-builds.html");
+                   throw Error("unable to start any build; either increase '--max-jobs' "
+                            "to also allow building locally, or ensure that the configured"
+                            "remote builder machines have all required system features."
+                            "For example, to enable the 'big-parallel' feature, use:"
+                            "\n    --builders 'yourbuilderhostname - - - - big-parallel'"
+                            "\nhttps://nixos.org/nix/manual/stable/advanced-topics/distributed-builds"
```

Comment:
This should instead refer to the features section in `man nix.conf` as we did for other hints in error messages recently.